dags/SQLalchemy_task/workers/Transaction_worker.py

```diff
@@ -14,14 +19,24 @@
 session = Session()

```

Comment:
please, consider **not creating** any kind of sessions separately from using the code - this is violating the side effect principle of func programming. 
https://dev.to/jamesrweb/principles-of-functional-programming-4b7c

**Example**: every import of this module creates an instance of session which is not closed later. 
Instead, consider using some kind of function with **try / finally** clause to create a session and close it on finish / failures. Then you can add this session as a parameter to the CRUD function

Reply:
also, this issue can be easily omitted using OOP or context managers (`with Session() as session`)